### DISCUSSION TITLE
package-lock.json unwanted changes (sorting/un-sorting of packages) [ > v 9.2 ]

### DISCUSSION THREAD

**Discussion by shumanhi** [npm/feedback#860] [CLI] [answered]
Our team is having an issue where running `npm install` sorts the packages listed inside `package-lock.json` when they are not sorted and does the opposite when they are sorted. Even when we all have the same version of NPM installed (or the latest). Anyone else is experiencing the same issue? 

This issue started after updating NPM version to 9.2. Prior to that, we never had this issue before.

**Comment by MylesBorins** [comment, marked as answer]
Hey @shumanhi. could you please open a bug report over at https://github.com/npm/cli, preferably with a minimal reproduction
